content/influxdb/clustered/get-started/setup.md

```diff
@@ -63,12 +63,12 @@ If stored at a non-default location, include the `--config` flag with each
 
     **Copy and paste the sample configuration profile code** into your `config.toml`:
 
-{{% code-placeholders "PORT|OAUTH_TOKEN_URL|OAUTH_DEVICE_URL" %}}
+{{% code-placeholders "PORT|HOST|OAUTH_TOKEN_URL|OAUTH_DEVICE_URL|OAUTH_CLIENT_ID" %}}
 ```toml
 [[profile]]
   name = "default"
   product = "clustered"
-  host = "{{< influxdb/host >}}"
+  host = "HOST"
```

Comment:
Host values are controlled by a global setting specified by the user. Code placeholders are scoped to a page and do not persist through a refresh. This should use the `influxdb/host` shortcode, not a code placeholder.

```suggestion
  host = {{< influxdb/host >}}
```

Reply:
ah that's interesting. Should we use global placeholders for other stuff too? For example some of the oauth stuff is referenced by multiple pages.

Reply:
No, anything that could contain a sensitive secret should not be stored. We don't want to be responsible in any way for securing people's secrets. The only global, user-defined value we currently store is the host/url.

Reply:
That makes sense! I'll revert this portion back

Reply:
The oauth config is not really a secret though. Is it still considered sensitive?

Reply:
ok; but just for the record; these are not credentials; they are just hostnames and IDs that used to configure the mechanism that will securely provide credentials.